doc/src/manual/performance-tips.md

```diff
@@ -863,7 +863,52 @@ type from an algorithm.  In the example above, we could have passed a `SubArray`
 Taken to its extreme, pre-allocation can make your code uglier, so performance measurements and
 some judgment may be required.   However, for "vectorized" (element-wise) functions, the convenient
 syntax `x .= f.(y)` can be used for in-place operations with fused loops and no temporary arrays
-([Dot Syntax for Vectorizing Functions](@ref)).
+(see the [dot syntax for vectorizing functions](@ref man-vectorized)).
+
+## More dots: Fuse vectorized operations
+
+Julia has a special [dot syntax](@ref man-vectorized) that converts
+any scalar function into a "vectorized" function call, and any operator
+into a "vectorized" operator, with the special property that nested
```

Comment:
Adjacent at the same level == nested, since  `a .+ b .* c == (+).(a, (*).(b,c))`.  Every nested call is fused until it hits a "non-dot" call; this is explained in `@ref man-vectorized`